projects/cashmere-examples/src/project-template/package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "project-template",
+  "version": "0.0.0",
+  "scripts": {
+    "ng": "ng",
+    "start": "ng serve",
+    "build": "ng build",
+    "test": "ng test",
+    "lint": "ng lint",
+    "e2e": "ng e2e"
+  },
+  "private": true,
+  "dependencies": {
+    "@angular/animations": "6.1.7",

```

Comment:
I am guessing this `package.json` is used to send to stack blitz. Does the angular versions need to match? If so is there a way we can match the root `package.json`? We may need to add documentation on how this works and what to do when we update package versions...

Reply:
good point.  I should probably have this file generated from the root `package.json`

Reply:
Just curious, why the `angular.json`?  In order to support new features added to the docs site?  The `angular.json` file in the StackBlitz won't necessarily have all the same config as the docs site, as it is only for StackBlitz.  I feel like changes to that should be more manual.

Reply:
I suppose we could auto-insert whatever is in the `"assets"`, `"styles"`, and `"scripts"` from the docs site project, though...

Reply:
I'm ok with the way it is